src/DataRequests/DataRequestFilterSets.jsx

```diff
@@ -0,0 +1,227 @@
+                                                    options={options}
+                                                    value={selected.value ? selected : null}
+                                                    isClearable={false}
+                                                    theme={overrideSelectTheme}
+                                                    placeholder='Select a filter set...'
+                                                    onChange={(e) => {
+                                                    values.filtersetId = e.value.id;
+                                                    setChangeFilterSetRequestError({
+                                                        isError: false,
+                                                        message: '',
+                                                    });
+                                                    setSelected(e);
+                                                    }}
+                                                    />
+                                                }
+                                            />
+                                            {selected.value && (
+                                            <SimpleInputField
+                                                label='Description'
+                                                hideLabel={true}
+                                                input={
+                                                    <textarea
+                                                    className={'filter-set-description'}
+                                                    id='open-filter-set-description'
+                                                    disabled
+                                                    placeholder='No description'
+                                                    value={selected.value?.description}
+                                                    />
+                                                }
+                                            />
+                                            )}
+                                        
+                                        {selected.value && (
+                                        <ExplorerFilterDisplay
+                                            filter={selected.value?.filter} title="Selected Filter Set"
+                                        />
+                                        )}
+                                    </div>
+                                )}
+                                
+                            </Field>
+                            
+                        </div>
+                        {selected.value && (
+                            <Button
+                                submit
+                                className='data-request__submit'
+                                label='Change Project Filter Set'
```

Comment:
"Change" feels kind of vague here since the operation appends a project to the list. Can we go back to "Add to Project" or something like that?

Reply:
I changed it because the way it currently works in the backend when you submit a new filter-set that filter-set is now the projects filter-set and removes the previous ones. I can switch this back but in the future I believe we will add the backend functionality to append new filter-sets and remove them etc..